web/themes/ventuno/src/scss/components/_navbar.scss

```diff
@@ -0,0 +1,315 @@
+.joinup-navbar {
+  background-color: $primary;
+  box-shadow: $box-shadow;
+  padding: 0;
+
+  @include media-breakpoint-between(xs, md) {
+    // height: 54px;
+  }
+
+  @include media-breakpoint-up(md) {
+    height: 72px;
+  }
+
+  .navbar-brand {
+    font-size: .75rem; // 12px
+  }
+
+  .navbar-brand-image {
+
+    @include media-breakpoint-between(xs, md) {
+      width: 6rem; // 96px
+    }
+
+    @include media-breakpoint-up(md) {
+      width: 8rem; // 128px
+    }
+  }
+
+  .navbar-motto {
+
+    @include media-breakpoint-between(xs, md) {
+      display: none;
```

Comment:
Use utility classes for this on the markup whenever possible. EX: d-none d-lg-block